### PR TITLE
ENGDOCS-1875

### PR DESCRIPTION
**Pull request by aevesdocker**
This PR adds all of the content updates for DD 4.26:

- Admins can set experimental and beta tabs on or off with Settings Management (Desktop Stability)
- minor reordering and rewording of settings (Desktop Stability)

- extension private marketplace visibility in toc (Publisher)

- kernelUDP setting (POS)
- Add admin setting for blocking 'docker load' (POS)

- Removes beta label from the 'Explore Builds' content

### CODE REVIEW

**Review by KatTomrushka — APPROVED** [review]
Looking good from DS POV!

File: data/toc.yaml [hunk 1]
```diff
@@ -1255,6 +1255,8 @@ Manuals:
           title: Manage non-Marketplace extensions
         - path: /desktop/extensions/settings-feedback/
           title: Change settings and give feedback
+        - path: /desktop/extensions/private-marketplace/
+          title: Configure a private marketplace (Beta)
```
Comment (aevesdocker) [inline data/toc.yaml:1259]:
```suggestion
          title: Configure a private marketplace (Experimental)
```

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 2]
```diff
@@ -113,6 +113,18 @@ The following `admin-settings.json` code and table provides an example of the re
     "locked": false,
     "sbomIndexing": true,
     "useBackgroundIndexing": true
+  },
+  "allowExperimentalFeatures": {
+    "locked": true,
+    "value": true
```
[the remark this answers is not in the document]
Comment (MihaelaStoica) [reply content/desktop/hardened-desktop/settings-management/configure.md:119]: I believe here we should give the example of value=false which is in sync with the "Turn off beta and experimental features" addition in _index.md

Also, all the other examples here have "locked": false. Shall this be the same, for consistency?

Also, for consistency, can we have the order of locked, value everywhere? as in
```
"x": {
    "locked":y,
    "value":z:
  },
```
for example"extensionsEnabled" and "blockDockerLoad" have it the other way around

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 3]
```diff
@@ -113,6 +113,18 @@ The following `admin-settings.json` code and table provides an example of the re
...
+  },
+  "allowBetaFeatures": {
+    "locked": true,
+    "value": true
```
Comment (MihaelaStoica) [inline content/desktop/hardened-desktop/settings-management/configure.md:123]: same here,  suggested "value": false

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 4]
```diff
@@ -135,6 +147,9 @@ The following `admin-settings.json` code and table provides an example of the re
 |`analyticsEnabled`|  |If `value` is set to false, Docker Desktop doesn't send usage statistics to Docker. |
 |`extensionsEnabled`|  |If `value` is set to false, Docker extensions are disabled. |
 |`scout`|| Setting `useBackgroundIndexing` to `false` disables automatic indexing of images loaded to the image store. Setting `sbomIndexing` to `false` prevents the manual indexing triggered by inspecting an image in Docker Desktop.<br><br>**Note**: Users can still use the `docker scout` CLI commands to index images, even if indexing is disabled in Settings Management. |
+| `allowExperimentalFeatures`| | If `locked` is set to `true`, users are no longer able to access or use experimental features.|
```
Comment (MihaelaStoica) [inline content/desktop/hardened-desktop/settings-management/configure.md:150]:
```suggestion
| `allowExperimentalFeatures`| | If `value` is set to `false`, the experimental features are disabled.|
```

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 5]
```diff
@@ -135,6 +147,9 @@ The following `admin-settings.json` code and table provides an example of the re
...
+| `allowBetaFeatures`| | If `locked` is set to `true`, users are no longer able to access or use beta features.|
```
Comment (MihaelaStoica) [inline content/desktop/hardened-desktop/settings-management/configure.md:151]:
```suggestion
| `allowBetaFeatures`| | If `value` is set to `false`, the beta features are disabled.|
```

File: content/desktop/hardened-desktop/settings-management/configure.md (hunk 4 above)
Comment (MihaelaStoica) [reply content/desktop/hardened-desktop/settings-management/configure.md:150]: As with the other admin settings, here we only need to describe what the setting controls, as the locking behaviour is described at the top of the page, line 46

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 6]
```diff
@@ -135,6 +147,9 @@ The following `admin-settings.json` code and table provides an example of the re
...
+| `blockDockerLoad` | | If `value` is set to `true`, users are no longer able to run [`docker load`](../../../engine/reference/commandline/load.md).|
```
Comment (MihaelaStoica) [inline content/desktop/hardened-desktop/settings-management/configure.md:152]: @gabriellavengeo please review this

**Review by MihaelaStoica — CHANGES_REQUESTED** [review]

File: content/desktop/hardened-desktop/settings-management/configure.md (hunk 6 above)
Comment (gabriellavengeo) [reply content/desktop/hardened-desktop/settings-management/configure.md:152]: Yeah, that's fine, or alternatively say that they should get an error if they try to

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**